src/core/MOM_CoriolisAdv.F90

```diff
@@ -239,9 +270,19 @@ subroutine CorAdCalc(u, v, h, uh, vh, CAu, CAv, OBC, AD, G, GV, US, CS, pbv, Wav
   is = G%isc ; ie = G%iec ; js = G%jsc ; je = G%jec
   Isq = G%IscB ; Ieq = G%IecB ; Jsq = G%JscB ; Jeq = G%JecB ; nz = GV%ke
   vol_neglect = GV%H_subroundoff * (1e-4 * US%m_to_L)**2
+  area_neglect = (1e-4 * US%m_to_L)**2
```

Comment:
The hard-coded area of 10^-8 m2 is not a negligible area compared with the area of a cell that might be used for small-scale process model.  This value should be made smaller.  (On the line above, there is a balance between making the area used in the volume small, but  not so small that vol_neglect would run into underflow issues if we were ever working with 32-bit reals.  The smallness of `vol_neglect` mostly comes from the smallness of `GV%H_subroundoff`.)